oat-sa/extension-tao-itemqti
# Fix/nex 90 fix moved required modules

Some SCSS files have been refactored and moved along with their component.
The qti default theme was expecting to import them but now they're loaded with the bundles or when requesting the component itself.

How to test : 
```
cd tao/views/build
npx grunt taoqtiitemsass
```
(fails without the PR) and I've taken a test to ensure the style is still accurate
